target/linux/ramips/dts/TL-MR3020V3.dts

```diff
@@ -0,0 +1,141 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,tl-mr3020-v3", "mediatek,mt7628an-soc";
+	model = "TP-Link TL-MR3020 v3";
+
```

Comment:
An alias node with `led-status` pointing to the power led is missing.

Reply:
I've added an alias node, as requested, yet this change request won't clear. Any advice?